src/components/authorPageComponents/Gallery/Gallery.js

```diff
@@ -13,10 +13,13 @@ class Gallery extends React.Component {
                 <h3 className={galleryStyles.gallery_title}>Gallery</h3>
                 {this.props.photosArrayOfObjects.map((edge) => {
                     const imageInfo = {};
+                    const delimiter = edge.description.search('x'); 
+                    const sizeLength = edge.description.length;
+
                     imageInfo.src = edge.file.url;
                     imageInfo.thumbnail = edge.file.url;
-                    imageInfo.thumbnailWidth = 320;
-                    imageInfo.thumbnailHeight = 150;
+                    imageInfo.thumbnailWidth = edge.description.substring(0,delimiter);
```

Comment:
`images[0].thumbnailWidth` of type `string`, expected `number`. Use sth like this `Number(edge.description.substring(0,delimiter))`